src/FormField/MultiLine.php

```diff
@@ -912,7 +912,7 @@ private function getDummyExpression($exprField, $model)
         preg_match_all('/\[[a-z0-9_]*\]|{[a-z0-9_]*}/i', $expr, $matches);
 
         foreach ($matches[0] as $match) {
-            $fieldName = substr($match, 1, -1);
+            $fieldName = mb_substr($match, 1, -1);
```

Comment:
no sense, this is not an operation on text, instead it removes ascii brackets (yes, op on text :D)